awx_collection/plugins/module_utils/tower_api.py

```diff
@@ -316,13 +316,12 @@ def make_request(self, method, endpoint, *args, **kwargs):
         elif kwargs.get('data'):
             self.url = self.url._replace(query=urlencode(kwargs.get('data')))
 
-        data = {}
+        data = None  # Important, if content type is not JSON, this should not be dict type
```

Comment:
@john-westcott-iv just to refresh you, this is the critical change that I hope fixes the real issue.